### ISSUE TITLE
[Feature] Return error if invalid query is provided

### ISSUE THREAD

**Issue by derN3rd** [ContentSquare/chproxy#220]
Currently when we send an invalid query (i.e. two data types in a comparison don't match) CHProxy just returns `400 Bad Request` and the exception code in the header.

Clickhouse itself sends 
```
Code: 53. DB::Exception: Types of column 1 in section IN don't match: UInt32 on the left, UInt64 on the right. (TYPE_MISMATCH) (version 22.7.1.2484 (official build))
```
in the response body, which I would like to have in CHProxy as well.

Is this already configurable?

Another possibility would be to use `log_debug: true` in the config, but that includes all requests, which is not a good idea in a production system. Maybe a new config option like `log_errors: true` which logs all failed queries could be helpful here.

Looking forward to other ideas an opinions.

Thanks in advance!

~derN3rd

**Comment by gontarzpawel** [comment]
Hello @derN3rd , what version of chproxy are you using? It looks like a bug that has been fixed in https://github.com/ContentSquare/chproxy/commit/05f4f59f599af00311011e18c84ebc17381c0eb9

included in the releases:
```
[v1.17.0](https://github.com/ContentSquare/chproxy/releases/tag/v1.17.0)  
[v1.16.4](https://github.com/ContentSquare/chproxy/releases/tag/v1.16.4) 
[v1.16.3](https://github.com/ContentSquare/chproxy/releases/tag/v1.16.3) 
[v1.16.2](https://github.com/ContentSquare/chproxy/releases/tag/v1.16.2)
```

**Comment by derN3rd** [comment]
> Hello @derN3rd , what version of chproxy are you using? It looks like a bug that has been fixed in [05f4f59](https://github.com/ContentSquare/chproxy/commit/05f4f59f599af00311011e18c84ebc17381c0eb9)
> 
> included in the releases:
> 
> ```
> [v1.17.0](https://github.com/ContentSquare/chproxy/releases/tag/v1.17.0)  
> [v1.16.4](https://github.com/ContentSquare/chproxy/releases/tag/v1.16.4) 
> [v1.16.3](https://github.com/ContentSquare/chproxy/releases/tag/v1.16.3) 
> [v1.16.2](https://github.com/ContentSquare/chproxy/releases/tag/v1.16.2)
> ```

Hey Garnek20,

Thanks for the hint. Our ansible deployment had the version hardcoded instead of using the latest, therefore I was checking with an older version.

Can confirm it's fixed now!
